src/importing/packaged-decks.md

```diff
@@ -1,18 +1,57 @@
 # Packaged Decks
 
+<!-- toc -->
+
+Anki packages (.apkg files) enable you to import decks, notes, notetypes, and cards from
+other users. They are commonly shared on [AnkiWeb](https://ankiweb.net/shared/decks).
+
+## Scheduling
+
+Anki packages may also contain scheduling information, which is useful if you want to
+transfer decks between devices or profiles. However, when importing
+a deck shared by someone else, you typically do not want to adopt their card intervals
+or review history.
+
+If you encounter imported cards with unexpectedly large intevals, the deck author may
+accidentally have included their scheduling information. You can use the
+[Set Due Date feature](../browsing.md#cards) to reset the imported cards. On Anki
+2.1.67 and later, you can remove any scheduling information during the import process.
+
 ## Updating
 
 When you import an .apkg file, Anki will identify any notes in it that are
 already in your collection due to a previous import. If the notes in the file
 are newer than your local copy, the notes will be updated with the contents of
-the file.
+the file by default.
 
-This updating process is not possible if the notetype is changed (e.g if either
+This updating process is generally not possible if the notetype is changed (e.g if either
 you or the deck author do things like add an extra field to the notetype).
 You will still be able to import any missing notes from the file, but
 notes you have imported previously will not be updated if the deck author
 has made changes.
 
+### Anki 2.1.67 and Later
+
+Anki 2.1.67 introduced more flexibility: You can choose to unconditionally
+update notes and notetypes, always overwriting your modifications,
+or, on the other hand, never update existing objects.
+
+Also, if both you and the deck author modified the same notetype, you can now decide to
+_merge_ the two versions. This will preserve all templates and fields contained by
```

Comment:
contained in

(for some reason GitHub won't let me make suggestions when viewing an individual commit)